bedrock/firefox/templates/firefox/lockwise/lockwise.html

```diff
@@ -0,0 +1,91 @@
+    <section class="mzp-c-hero mzp-has-image">
+      <div class="mzp-l-content">
+        <div class="mzp-c-hero-body">
+          <img src="{{ static('img/firefox/lockwise/lockwise-horizontal.svg') }}" class="lockwise-logo" />
+          <h1 class="mzp-c-hero-title">{{ _('Take your passwords everywhere') }}</h1>
+          <div class="mzp-c-hero-desc">
+            <p>{{ _('Securely access the passwords you’ve saved in Firefox from anywhere — even outside of the browser.') }}</p>
+          </div>
+          <div class="mobile-download-buttons">
+            <a href="{{ lockwise_adjust_url('ios', 'accounts-page') }}" data-link-type="download" data-download-os="iOS" class="ios-button">
+              <img src="{{ l10n_img('firefox/ios/btn-app-store.svg') }}" alt="{{ _('Download on the App Store') }}" width="152" height="45">
+            </a>
+            {{ google_play_button(href=lockwise_adjust_url('android', 'accounts-page')) }}
+          </div>
+          <div class="mzp-c-hero-cta add-on-download">
+            <div class="mzp-c-button-download-container">
+              <p>{{ _('Try the Lockwise add-on') }}</p>
+              <a href="#" class="mzp-c-button mzp-t-product">{{ _('Install for Firefox') }}</a>
+            </div>
+          </div>
+          <div class="mzp-c-hero-image">
+          {% if LANG.startswith('en-') %}
+            <img src="{{ static('img/firefox/lockwise/EN.svg') }}" alt="">
+          {% elif LANG == 'de' %}
+            <img src="{{ static('img/fmake irefox/lockwise/DE.svg') }}" alt="">
+          {% elif LANG == 'fr' %}
+            <img src="{{ static('img/firefox/lockwise/FR.svg') }}" alt="">
+          {% elif LANG == 'it' %}
+            <img src="{{ static('img/firefox/lockwise/IT.svg') }}" alt="">
+          {% elif LANG.startswith('es-') %}
+            <img src="{{ static('img/firefox/lockwise/ES.svg') }}" alt="">
+          {% else %}
+            <img src="{{ static('img/firefox/lockwise/EN.svg') }}" alt="">
+          {% endif %}
+          </div>
+        </div>
+      </div>
+      <div class="mzp-c-hero-image">
+        <img src="" alt="">
+      </div>
+    </section>
+    <section class="dark-content">
+      <section class="mzp-l-card-third mzp-l-content">
+        <section class="mzp-c-card-picto ">
+          <div class="mzp-c-card-picto-content encryption">
+            <p class="picto-content">{{ _('256-bit encryption protects you while syncing') }}</p>
+          </div>
+        </section>
+        <section class="mzp-c-card-picto">
+          <div class="mzp-c-card-picto-content touch">
+            <p class="picto-content">{{ _('Get to your passwords securely with Face or Touch ID') }}</p>
+          </div>
+        </section>
+        <section class="mzp-c-card-picto">
+          <div class="mzp-c-card-picto-content privacy">
+            <p class="picto-content"><span>{{ _('Your privacy comes first.')}}</span><span>{{ _('We keep your data safe, never sold.') }}</span></p>
```

Comment:
Drive-by nit: This renders without a space between the two strings: "…first.We…"